docs/src/pages/components/geo/index.page.mdx

```diff
@@ -6,23 +6,76 @@ description: Amplify UI Geo provides UI components for maps and location search
 import { Example } from '@/components/Example';
 import { Feature } from '@/components/Feature';
 import { Fragment } from '@/components/Fragment';
-import { XStateInspector } from '@/components/XStateInspector';
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
 
 Amplify UI Geo provides UI components for maps and location search for popular front end frameworks, powered by
 [Amplify Geo APIs](https://docs.amplify.aws/lib/geo/getting-started/q/platform/js/) and
 [Amazon Location Service](https://aws.amazon.com/location/).
 
+### Quick Start
+
+> Prerequisite: [Create a Geo backend](https://docs.amplify.aws/lib/geo/getting-started/q/platform/js/)
+
+## AmplifyMap
+
+The `AmplifyMap` component adds an interactive map to your application.
+
+`AmplifyMap` is fully integrated with the open source library [react-map-gl](https://visgl.github.io/react-map-gl/) v7
+while using [maplibre-gl-js](https://maplibre.org/maplibre-gl-js-docs/api/) as the map tile source. `AmplifyMap` is
+used as a replacement to `react-map-gl`'s [default map](https://visgl.github.io/react-map-gl/docs/api-reference/map)
+and supports the same functionality.
+
+Install the required peer dependencies using your preferred package manager. Below, we use [Yarn](https://yarnpkg.com/)
+as an example:
+
+```shell
+yarn add maplibre-gl-js@1 react-map-gl
```

Comment:
Is `@1` necessary?

Reply:
Looks like that matches the official documentation here too, so the @1 needs to be there. https://docs.amplify.aws/lib/geo/maps/q/platform/js/